Management/src/ActuatorWeb/Program.cs

```diff
@@ -0,0 +1,82 @@
+using Steeltoe.Common;
+using Steeltoe.Configuration.CloudFoundry;
+using Steeltoe.Management.Endpoint;
+using Steeltoe.Management.Endpoint.Actuators.All;
+using Steeltoe.Management.Endpoint.SpringBootAdminClient;
+using Steeltoe.Management.Prometheus;
+using Steeltoe.Samples.ActuatorWeb;
+using Steeltoe.Samples.ActuatorWeb.Pages;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+
+// Steeltoe: Add Cloud Foundry configuration provider and options classes.
+builder.AddCloudFoundryConfiguration();
+
+// Steeltoe: Add all actuators, specifying an authorization policy for use outside the Cloud Foundry context.
+builder.Services.ConfigureActuatorEndpoints(configureEndpoints =>
+{
+    if (!Platform.IsCloudFoundry)
+    {
+        configureEndpoints.RequireAuthorization("actuator.read");
+    }
+});
+builder.Services.ConfigureActuatorAuth();
+builder.Services.AddAllActuators();
+builder.Services.AddPrometheusActuator();
+if (builder.Configuration.GetValue<bool>("UseSpringBootAdmin"))
+{
+    builder.Services.AddSpringBootAdminClient();
+}
+
+builder.Services.ConfigureOpenTelemetry(builder.Configuration);
+
+// Steeltoe: Register HttpClients for communicating with a backend service.
+builder.Services.AddHttpClient<WeatherModel>(SetBaseAddress);
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+    app.UseHsts();
+}
+
+app.UseHttpsRedirection();
+
+app.UseStaticFiles();
+
+app.UseRouting();
+
+app.UseAuthorization();
+
+app.MapRazorPages();
+
+app.UsePrometheusActuator();
+
+app.Run();
+return;
+
+static void SetBaseAddress(IServiceProvider serviceProvider, HttpClient client)
```

Comment:
I wish somebody built a system to take care of this...

All kidding aside, a comment explaining this code exists to keep the sample simple, but the prefered way to deal with this is service discovery (with a link to the docs) would be nice.